app/scripts/modules/core/securityGroup/securityGroup.read.service.js

```diff
@@ -169,6 +169,7 @@ module.exports = angular.module('spinnaker.core.securityGroup.read.service', [
           }
         }
       }
+      data = _.uniq(data);
```

Comment:
...probably because there's no deduping on #L98 or #L125, so the `filter` calls are not going to matter.